base/server/python/pki-healthcheck/pkihealthcheck/core/main.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2019 FreeIPA Contributors see COPYING for license
+#
+
+import logging
+import sys
+
+from ipahealthcheck.core.core import RunChecks
+
+logging.basicConfig(format='%(message)s')
+logger = logging.getLogger()
+
+
+def main():
+    checks = RunChecks(['pkihealthcheck.registry'],
```

Comment:
Just a question here: It seems we have one single config file for an entire pki installation on a given host. Is there any reason to have config files for each instance, in the case there are multi instances on the machine?

Reply:
Same as [this](https://github.com/dogtagpki/pki/pull/301#discussion_r355564672) comment?